src/zenml/utils/yaml_utils.py

```diff
@@ -102,6 +102,20 @@ def is_yaml(file_path: str) -> bool:
     return False
 
 
+def comment_out_yaml(yaml_string: str) -> str:
+    """Comments out a yaml string.
+
+    Args:
+        yaml_string: The yaml string to comment out.
+
+    Returns:
+        The commented out yaml string.
+    """
+    lines = yaml_string.splitlines(keepends=True)
+    lines = ["# " + line for line in lines]
+    return "".join(lines)
+
+
```

Comment:
where is this needed? :D

Reply:
I have a method that generates you a `config.yaml` that can be specified in `pipeline.run(...)`. And by default I just comment out the entire yaml file so users can enable only the things they want to configure.